public/locales/en/translation.json

```diff
@@ -286,5 +286,194 @@
+      "SilverGray": "Silver gray"
+    },
+    "eyebrowType": {
+      "label": "Eyebrow type",
+      "Angry": "Angry",
+      "AngryNatural": "Angry 2",
+      "Default": "Default",
+      "DefaultNatural": "Default 2",
+      "FlatNatural": "Flat",
+      "FrownNatural": "Frown",
+      "RaisedExcited": "Excited",
+      "RaisedExcitedNatural": "Excited 2",
+      "SadConcerned": "Concerned",
+      "SadConcernedNatural": "Concerned 2",
+      "UnibrowNatural": "Unibrow",
+      "UpDown": "Raised",
+      "UpDownNatural": "Raised 2"
+    },
+    "facialHairColor": {
+      "label": "Facial hair color",
+      "Auburn": "Auburn",
+      "Black": "Black",
+      "Blonde": "Blonde",
+      "BlondeGolden": "Golden blonde",
+      "Brown": "Brown",
+      "BrownDark": "Dark brown",
+      "Platinum": "Platinum",
+      "Red": "Red"
+    },
+    "facialHairType": {
+      "label": "Facial hair type",
+      "Blank": "Blank",
+      "BeardMedium": "Medium beard",
+      "BeardLight": "Light beard",
+      "BeardMajestic": "Majestic beard",
+      "MoustacheFancy": "Fancy moustache",
+      "MoustacheMagnum": "Magnum moustache"
+    },
+    "skinColor": {
+      "label": "Skin color",
+      "Tanned": "Tanned",
+      "Yellow": "Yellow",
+      "Pale": "Pale",
+      "Light": "Light",
+      "Brown": "Brown",
+      "DarkBrown": "Dark brown",
+      "Black": "Black"
+    },
+    "eyeType": {
+      "label": "Eye type",
+      "Close": "Closed",
+      "Cry": "Crying",
+      "Default": "Default",
+      "Dizzy": "Dizzy",
+      "EyeRoll": "Rolling eyes",
+      "Happy": "Happy",
+      "Hearts": "Hearts",
+      "Side": "Look sideways",
+      "Squint": "Squint",
+      "Surprised": "Surprised",
+      "Wink": "Wink",
+      "WinkWacky": "Wacky wink"
+    },
+    "mouthType": {
+      "label": "Mouth type",
+      "Concerned": "Concerned",
+      "Default": "Default",
+      "Disbelief": "Disbelief",
+      "Eating": "Eating",
+      "Grimace": "Grimace",
+      "Sad": "Sad",
+      "ScreamOpen": "Screaming",
+      "Serious": "Serious",
+      "Smile": "Smile",
+      "Tongue": "Tongue out",
+      "Twinkle": "Twinkle",
+      "Vomit": "Vomit"
+    },
+    "topType": {
+      "label": "Top type",
+      "Eyepatch": "Eyepatch",
+      "Hat": "Hat",
+      "Hijab": "Hijab",
+      "LongHairBigHair": "Long, wavy hair",
+      "LongHairBob": "Bob",
+      "LongHairBun": "Bun",
+      "LongHairCurly": "Long, curly hair",
+      "LongHairCurvy": "Long, curvy hair",
+      "LongHairDreads": "Dreadlocks",
+      "LongHairFrida": "Frida Kahlo",
+      "LongHairFro": "Afro",
+      "LongHairFroBand": "Afro with headband",
+      "LongHairMiaWallace": "Mia Wallace",
+      "LongHairNotTooLong": "Medium length hair",
+      "LongHairShavedSides": "Long hair with sidecut",
+      "LongHairStraight": "Straight, long hair 1",
+      "LongHairStraight2": "Straight, long hair 2",
+      "LongHairStraightStrand": "Straight, long hair 3",
+      "NoHair": "No hair",
```

Comment:
Auf der [avataaars-Seite](https://getavataaars.com) sind ab und zu Optionen ausgeblendet, z.B. die Haarfarbe wenn man gerade eine Glatze hat. Was hältst du davon, das hier auch einzubauen? Oder eher mal später in einem neuen Ticket?